package.json

```diff
@@ -13,8 +13,9 @@
     "build": "webpack --mode production"
   },
   "dependencies": {
+    "@kno2/eslint-config": "^1.0.8",
     "autoprefixer": "^9.4.9",
-    "bluebutton": "github:Royal-Jay/bluebutton.js#0.5.0-rc.6",
+    "bluebutton": "github:djbreen7/bluebutton.js#KEW-329",
```

Comment:
@djbreen7 NOTE TO SELF:

- Create new bluebutton release
- Update package.json to use Kno2/bluebutton.js#{release}